quiz/backend/config/urls.py

```diff
@@ -13,11 +13,22 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.conf import settings
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 # TODO: change admin endpoint for production and reverse proxy
 
 urlpatterns = [
     path('admin/', admin.site.urls),
 ]
+
+if settings.DEBUG:
+    try:
```

Comment:
Nie widzę sensu odpalać try, my to mamy w projekcie, więc musi się zaimportować. Simple is better than complex™

Reply:
Wrzuciłem te paczki jako dev dependencies i produkcyjny docker rusza bez tych paczek
`RUN poetry install --without dev --sync`, więc bez try/except się wywali

Reply:
Zmienione zgodnie z rozmową